### ISSUE TITLE
Enhancement suggestion: Stix Two Math Bold

### ISSUE THREAD

**Issue by callegar** [stipub/stixfonts#211]
I'd like to suggest a bold math version like XITS has.

**Comment by Griger** [comment]
Yes, I think that this feature could make STIX Two a more awesome font. Sometimes you want to highlight a text that contains math symbols, and since there is not a STIX Two Math bold version, this text has some visual inconsistencies.

Many thanks for your time and your font. :smile:

**Comment by tiroj** [comment]
The trouble with this is the use of bold styled alphanumeric symbols within mathematical typesetting, which means that if the base style is set to typographic bold, then all the bold styled characters within an equation need to be made super bold in order to maintain their distinctive weight styling relative to the other letters and symbols. This is doable, but it means a bold math font is a _lot_ of additional work: basically every glyph needs to be adjusted to maintain relative weight.

**Comment by callegar** [comment]
Maybe this bold-within-bold part could be left as a second step. At least the issue of having a short bit of a formula in a heading or in an abstract section where some journal or conference conventions impose the usage of bold fonts would be partially solved. As it is now, you get the "abrupt jump" out of bold for the formula that stands out in a very unpleasant way and the only way to limit the issue is to fake a bold style (which is still quite bad since it changes the character size and the baseline). In the end the formula bits that get in some text region where the base style is bold are typically rather short, meaning that the probability of getting a mathbold in a base bold style would likely be rather low.

**Comment by davidmjones** [comment]
I second @tiroj 's comments and would add that if, say, bold non-alphabetic symbols and superbold alphabetic symbols become common, it's only a matter of time before mathematicians co-opt them for use alongside their non-bold counterparts to mean something different.

In mathematical notation, the weight and style of the characters already carry inherent semantic content.  Even the Unicode committee had to accept that -- that's why we have the various Mathematical Alphanumeric Symbols in plane 1.  IMO, it's long past time for publishers to realize that you can't just say "bold this formula" and expect to get anything semanitcally coherent out of it.

[In case it's not clear, I'm speaking here for myself, not for the STI Pub Committee.]

**Comment by callegar** [comment]
I understand the point, but I cannot see how not making a bold variant can help preventing people from co-opting the bold math version to associate some semantics to a symbol or maybe some semantic variant to a symbol (e.g. using ***V*** for a vector whose entries are *Vi* with index i). This is something that is already done and that already has the potential to cause trouble when math is placed inside a boldened text area (e.g. an abstract or a section heading) — either you get ambiguity or the weird look of a font weight change. The only difference is that as of today you need to stick to XITS or to fake bold and the problem is that you stick to XITS or to fake bold even for usages of a bold math version when there are the premises for it not to lead to significant notation ambiguity.

**Comment by apoorvpotnis** [comment]
> The trouble with this is the use of bold styled alphanumeric symbols within mathematical typesetting, which means that if the base style is set to typographic bold, then all the bold styled characters within an equation need to be made super bold in order to maintain their distinctive weight styling relative to the other letters and symbols. This is doable, but it means a bold math font is a _lot_ of additional work: basically every glyph needs to be adjusted to maintain relative weight.

So are there any plans for this in the future?

Only a couple of free math fonts have the bold math variant. Kp-math and XITS full support, while XCharter and Erewhon have partial support. A bold math font would really enhance the already beautiful STIX2 fonts.
